[IDE] Console view not working locally in Safari

**Describe the bug**
Console view in XSK run locally in Docker returns an error `[ERROR] Connection problem! Check security roles assignments.`

> What version of the XSK are you using?
0.9.2

**To Reproduce**
Steps to reproduce the behavior:
1. Run xsk:0.9.2 locally
2. Open Safari
3. Open XSK
4. See error

**Expected behavior**
Logs are streamed and the error is not shown

**Desktop (please complete the following information):**
 - OS: MacOS 11.6
 - Browser: Safari
 - Version: 15.0

--- a/releng/server/src/main/webapp/WEB-INF/web.xml
+++ b/releng/server/src/main/webapp/WEB-INF/web.xml
@@ -124,7 +124,12 @@
 	</resource-ref> -->
 	
 	<login-config>
-        <auth-method>BASIC</auth-method>
+        <auth-method>FORM</auth-method>
+        <realm-name>default</realm-name>
+        <form-login-config>
+          <form-login-page>/login-form.html</form-login-page>
+          <form-error-page>/login-error.html</form-error-page>
+        </form-login-config>
     </login-config>
 
 	<!-- Error Pages -->     

